### DISCUSSION TITLE
Include Playground Source Code

### DISCUSSION THREAD

**Discussion by gbaron** [dm4t2/vue-currency-input#264] [Q&A] [answered]
Hey!
Thank you very much for building this package, it's very helpful. 
I would like to suggest that the code for the Playground (from the Docs website) will be include in this project. I think it will be very helpful. 

Thanks!

**Comment by dm4t2** [comment, marked as answer]
You can find the source here: https://github.com/dm4t2/vue-currency-input/blob/master/docs/.vitepress/theme/components/Demo.vue

**Reply by gbaron** [reply]
Thank you!
